presto-main/src/main/java/com/facebook/presto/sql/planner/plan/AggregationNode.java

```diff
@@ -186,6 +187,12 @@ public AggregationNode(@JsonProperty("id") PlanNodeId id,
         return groupingSets;
     }
 
+    public boolean hasDefaultOutput()
```

Comment:
@martint and I have disagreed on where these helper functions should be in the past - he doesn't like them to be in the PlanNodes. I don't remember the reasoning now, though.